docs/DesignDocs/ExtendedEnclaveInitializationData.md

```diff
@@ -25,7 +24,7 @@ User Experience
 
 The new API is experimental and details are subject to change, but the examples given here give an idea of the added functionality.
 
-We recommend that all parameters specified at enclave signing time should be set to a value that would prevent the enclave from loading, typically 0. In the case of SGX that means `NumStackPages=0`, `NumHeapPages=0`, and `NumTCS=0`.
+We recommend that all parameters specified at enclave signing time should be set to a value that would prevent the enclave from loading, typically 0. In the case of SGX that means `NumStackPages=0`, `NumHeapPages=0`, and `NumTCS=1`.
```

Comment:
I think it might help to pull the reasoning about `NumTCS=1` forward to this paragraph, as the last section is really just a list of verification steps.

Reply:
The EEID reports verification process is the context of the security consideration of NumTCS choice. The latest pull request version mentions the security consideration in the early section and refers the readers to the later section for details.